### PR TITLE
#4813 - Remove stars if disabled

### PR DESCRIPTION
**Pull request by IrinaZhadzinets**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/4813

**Problem:**
* Stars and rate 0.00 are displayed in review tab on PDP if all ratings are disabled or not assigned to the store

**In this PR:**
* Don't show not painted stars and rate 0.00

### CODE REVIEW

File: packages/scandipwa/src/component/ProductReviews/ProductReviews.component.js [hunk 1]
```diff
@@ -79,11 +81,13 @@ export class ProductReviews extends PureComponent {
...
                     review_count
                 } = {}
-            }
+            },
+            reviewRatings
         } = this.props;
 
         const STARS_COUNT = 5;
         const PERCENT = 100;
+        const SHOW_STARS = reviewRatings.length > 0;
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/ProductReviews/ProductReviews.component.js:90]: This is a variable, not a constant. Please make it camelCase.

**Comment by aleksandrakorolova** [comment]
Please, don't show Stars and rate 0.00 on PDP if some of the ratings are enabled now but it was disabled when user left review
Screenshot 2022-07-26 at 10 46 35

**Comment by aleksandrakorolova** [comment]
1. Fixed
2. Stars and Rate are displayed on PDP if some of the ratings are disabled now but it was enabled when user left review 
Screenshot 2022-08-09 at 16 16 10
**Expected result:**
Show Number of reviews only
Screenshot 2022-08-09 at 16 16 38

**Comment by aleksandrakorolova** [comment]
1 and 2 issues :

- Rates are inactivated:
 --Product has review with ratings - **Please, show rate and stars** 
 --Product has at least one review without rating - **Works as expected**
 
 - Rates are invisible in this store:
  --Product has review with ratings - **Works as expected**
 --Product has at least one review without rating - **Can reproduce**
  https://scandipwapmrev.indvp.com/default/bag-empty-simple.html
Screenshot 2022-08-25 at 13 11 49

Screenshot 2022-08-25 at 11 00 31

3. Ooops page appears when opening product with review - **Fixed after the last commit** 
Steps:
FE:
- Open PDP 
- Left review

Admin panel:
- Open Marketing -> All reviews
- Open this review
- Approve it

FE:
- Open the same PDP without rating -  https://pr-5055.d25kd40decjpue.amplifyapp.com/default/1-simple-product.html or https://pr-5055.d25kd40decjpue.amplifyapp.com/default/bundle-product-with-not-required-radio-buttons.html
- Open product with review with rating

**Actual result:**
Ooops page opens
Error message 'TypeError: Cannot read properties of undefined (reading 'forEach')' in the console
Screenshot 2022-08-25 at 09 44 26

**Comment by AleksandrsKondratjevs** [comment]
Closing due to deprecated code and no activity.

**Pull request closed without merging**